### ISSUE TITLE
 Client Handshake failed. The client may not trust Selenium Wire's certificate

### ISSUE THREAD

**Issue by pineapple696969** [wkeeling/selenium-wire#214]
I'm using selenium wire and I already installed the certificate and the connection is secure and proxies are working but im getting this in the console:

 127.0.0.1:61519: Client Handshake failed. The client may not trust Selenium Wire's certificate for static.nike.com

here's my code:
 https://dpaste.com/AC57BUZEJ

thanks

**Comment by wkeeling** [comment]
Thank you for raising this. Does this warning happen if you use Selenium Wire without the proxies?

**Comment by pineapple696969** [comment]
I tried and it doesn't? What certificate do I need to install and import?

**Comment by wkeeling** [comment]
If you've imported Selenium Wire's `ca.crt` that should be enough. There may be some additional information that we're missing because it gets logged at debug level. Are you able to increase the log level to DEBUG and see whether there's anything else coming through after the warning message?

**Comment by pineapple696969** [comment]
I tried to do that but nothing's showing up or maybe I'm not doing it right cause im new. But here's everything going in on console.
[warning.txt](https://github.com/wkeeling/selenium-wire/files/5977737/warning.txt)

**Comment by pineapple696969** [comment]
Got a work around on the logging. This what came up. 
[logfilename.log](https://github.com/wkeeling/selenium-wire/files/5978479/logfilename.log)

**Comment by wkeeling** [comment]
Thanks for sharing the logging. The debug message immediately after the warning indicates that the error is 
```
SysCallError(10053, \'WSAECONNABORTED\')
```

That's a fairly general message indicating that the connection has dropped and can have many different causes. The proxy server is obviously playing a role here though. As far as you are aware, is it having any negative impacts on page loading - e.g. do parts of the page fail to load?

**Comment by pineapple696969** [comment]
No, but it's spamming the console with selenium's certificate may not be trusted

**Comment by wkeeling** [comment]
I've created a new release 4.0.5 that downgrades the warning to a debug message. I appreciate it's not a fix, but it should at least quieten the console output whilst I investigate what's causing the message.

**Comment by pineapple696969** [comment]
I also did set the level to error yesterday while i'm waiting for a fix. thank you!
